src/server/server_family.cc

```diff
@@ -1287,6 +1287,10 @@ GenericError ServerFamily::DoSave(bool ignore_state) {
 
 GenericError ServerFamily::DoSave(bool new_version, string_view basename, Transaction* trans,
                                   bool ignore_state) {
+  if (shard_set->IsTieringEnabled()) {
```

Comment:
What about scheduled save? Do you intentionally allow that?

Reply:
schedule save also calls DoSave. am I missing something here?

Reply:
Nope, my bad :)
I would however `exit(-1)` (or at least print an error) inside `ServerFamily::SnapshotScheduling()` if `IsTieringEnabled()`, as they make no sense together